lib/otel/segments/consumer.js

```diff
@@ -37,9 +38,11 @@ function createConsumerSegment(agent, otelSpan) {
   //   'host',
   //
   // )
-  transaction.name = segmentName
+  transaction.setPartialName(segmentName)
+  transaction.finalizeName()
 
   const segment = agent.tracer.createSegment({
+    recorder,
     name: segmentName,
```

Comment:
based on the message-shim/subscribe-consume. this needs to be `transaction.getFullName()`